integration/shareddata/faker.go

```diff
@@ -15,22 +15,213 @@
 package shareddata
 
 import (
-	"fmt"
+	"math"
 	"math/rand"
+	"strconv"
 	"time"
 
-	fakerlib "github.com/jaswdr/faker"
```

Comment:
That library uses a ton of memory allocations. It dominates everything else in the benchmark's memory profile. And we don't need many features from it.